rfd/0029-account-lifecycle.md

```diff
@@ -56,8 +56,6 @@ domain names in the prefix about leaked tokens.
 * [ ] Each token can only be used one time successfully or unsuccessfully.
 * [ ] Teleport should store tokens hashed with `crypto/bcrypt` and compare using constant-time compare.
 Teleport should store the time when the tokens were generated alongside the hashed tokens.
-* [ ] Trying a wrong token 3 times in a row should lead to a temporary account lock following
```

Comment:
@klizhentas or @kimlisa, do you remember the reasoning behind this requirement?

Did we have brute-force concerns with some of the recovery flows? Say, leaked token + guessed password to register a new MFA device, or something like that?